### DISCUSSION TITLE
Текущее стояние Ru Core

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#220] [General]
Коллеги,
в связи с тем, что теперь все артефакты RuCore собраны в одном узле сайта - [Руководство Ru Core](http://fhir-ru.zendoc.me/core), предлагаю провести ревизию текущего состояния руководства и систематизировать задачи по его доработке до версии, которую можно вынести на публичное обсуждение. 
Видимо, будут задачи на трех уровнях:
* экспертные - например, руководство сегодня не отражает российские требования к учету оказанной медицинской помощи; решения о профилировании ресурсов для диагностики приняты на уровне руководства СЭМД ПЛИ, а не RuCore и т.д.
* редакторские - например, не все системы кодирования и наборы значений оформлены по правилам; справочники не добавлены на сайт и поэтому не используются при валидации, и т.д.
* инфраструктурные - например, детальное описание атрибутов ресурса хорошо бы показывать на отдельной вкладке, differential view не подсвечивает, что именно было изменено, и т.д.

**Comment by Vrodvi** [comment]
[редакторские]

1. Несмотря на то что всё собрано, навигация по содержимому руководства крайне затруднительно. Вот некоторые замеченные сложности:

1.1 Битые ссылки, которые никуда не ведут, например:
https://nsi.rosminzdrav.ru/#!/oidRepository/refbook/1.2.643.5.1.13.13.11.1002/version/7.6
тут http://fhir-ru.zendoc.me/core.valuesets.core-vs-medical-and-pharmaceutical-workers-positions#:valueset-zen-definition
![image](https://user-images.githubusercontent.com/92928137/212008047-de3c21c5-42d1-48a7-bed7-7802c5e47200.png)

1.2 Поиск по сайту не работает, например:
невозможно найти справочник по OIDу со страницы набора значений:
![image](https://user-images.githubusercontent.com/92928137/212008563-ad56b824-90eb-4be9-8c95-d5a4542b7e33.png)

1.2.1 На этом же примере - нет прямой ссылки между набором значений и справочником, из которого взяты значения, поэтому и приходится использовать поиск по сайту (который не работает):
http://fhir-ru.zendoc.me/core.codesystems.cs-nsi-medical-and-pharmaceutical-workers-positions
http://fhir-ru.zendoc.me/core.valuesets.core-vs-medical-and-pharmaceutical-workers-positions

12.1.2 Нужны связи между наборами значений и профилями ресурсов, которые их используют (и в обратную сторону)

----

2. Пустые разделы с непонятным предназначением и неговорящими названиями:
![image](https://user-images.githubusercontent.com/92928137/212009697-fb8cf970-0e42-4427-9ec0-6345082bdb69.png)

**Comment by Vrodvi** [comment]
[инфраструктурные]

1. В руководстве не соблюдаются рекомендации, которые оно же само и выставляется:

1.1 В наборах значений не используется URL, который отмечен как предпочтительный для использования:
![image](https://user-images.githubusercontent.com/92928137/212012012-1c8bb9dc-66af-4de8-b7c4-e5c4c8b8d268.png)

----

2. В связи с тем, что не проводится ревью сведений, размещённых в руководстве, возможны концептуальные ошибки, противоречащие стандарту, например: 
в качестве URL для системы кодирования указан URL его набора значений:
`http://fhir.ru/core/vs/core-vs-medical-and-pharmaceutical-workers-positions`
вот тут - http://fhir-ru.zendoc.me/core.codesystems.cs-nsi-medical-and-pharmaceutical-workers-positions
![image](https://user-images.githubusercontent.com/92928137/212012891-64fde57c-5b9f-4873-abf7-4bbca6df0db4.png)

**Comment by Vrodvi** [comment]
[инфраструктурные]

2. Некоторые наборы значений вообще не имеют связанных с ними систем кодирования или NamingSystem'ов 
![image](https://user-images.githubusercontent.com/92928137/212013587-e91633cb-ec5d-4559-8bc6-2e06b856a1ca.png)
Ситуация усугубляется:
- тем, что между системой кодирования и набором значений не указываются ссылки
- тем, что между набором значений и профилями не указываются ссылки, поэтому не возможно понять используется ли вообще где-то этот набор значений или он просто лишний и его забыли с сайта убрать
- тем, что на сайте не работает поиск
